.github/workflows/ci.yml

```diff
@@ -139,6 +149,27 @@ jobs:
     - run: yarn --frozen-lockfile
 
     - run: sudo apt-get install xvfb
-
     - name: Run smoke tests
       run: xvfb-run --auto-servernum yarn smoke --debug -j=1 --retries=2 --invert-match ${{ matrix.smoke-test-invert }} $SMOKE_GROUP_1
+  
+  # Only run smoke tests for windows against stable chrome.
+  smoke-windows:
+    strategy:
```

Comment:
probably don't need strategy here for something that isn't a matrix, no? at least update the comments for why we really need it if we still do